### PR TITLE
add gps_device

### PR DESCRIPTION
**Pull request by renan028**
**Description**
Add GPS device for webots_ros2 API. 

**Affected Packages**
List of affected packages:
  - webots_ros2_core

**Additional context**
You can test it by adding:
```
GPS {
      accuracy 0.5
      speedNoise 2
    }
```
In the tesla model and creating a subscriber type `PointStamped`, topic `gps/gps`

### CODE REVIEW

File: webots_ros2_core/webots_ros2_core/devices/gps_device.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        else:
+            self.__gps_publisher = node.create_publisher(
+                PointStamped, self._topic_name + '/gps', qos_profile_sensor_data)
+
+    def step(self):
+        stamp = super().step()
+        if not stamp:
+            return
+
+        if self.__gps_publisher.get_subscription_count() > 0 or self._always_publish:
```
Comment (lukicdarkoo) [inline webots_ros2_core/webots_ros2_core/devices/gps_device.py:79]:
```suggestion
        if self.__gps_publisher.get_subscription_count() > 0 or self.__speed_publisher.get_subscription_count() > 0 or self._always_publish:
```
If a user subscribes to `/speed` and not to `/gps` it should also work

**Review by lukicdarkoo — CHANGES_REQUESTED** [review]
Nice! Make sure that we use a maximum line length of 128, not the default line length.

**Review by lukicdarkoo — APPROVED** [review]
Thank you!

We should consider exposing a velocity vector-like here:
http://wiki.ros.org/hector_gazebo_plugins#GazeboRosGps
(this has to be done in the Webots core)

**Comment by renan028** [comment]
Yes! I agree.

**Pull request merged**